package task_15_ExceptionsAndAssertions;

import java.math.BigInteger;

public class Test {

    public static void main(String[] args) {

        try {
            System.out.println("Catch me if you can!");

        } finally {
            System.out.print("Here you are!");
        }


    }

}
